src/index.html

```diff
@@ -268,5 +279,15 @@ <h1 class="h-header h-txt24">DOŚWIADCZENIE</h1>
 
         </div>
     </div>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-126028127-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-126028127-1');
+    </script>
```

Comment:
Only for trades

Reply:
What do you mean?

Reply:
After InnoTrans we need to remove it or we need to add Cookie Popup